packages/contracts/contracts/DependencyRegistryV0.sol

```diff
@@ -856,6 +932,129 @@ contract DependencyRegistryV0 is
         }
     }
 
+    /**
+     * @notice Adds a full license text to license `_licenseType`, by way of
+     *         providing a string to write to bytecode storage.
+     * @param _licenseType Name of license type (i.e. "MIT") used to identify license.
+     * @param _text Text to be added. Required to be a non-empty string,
+     *                but no further validation is performed.
+     */
+    function addLicenseText(
+        bytes32 _licenseType,
+        string memory _text
+    ) external {
```

Comment:
lets bring these external functions (external view are fine) up to the block of external functions, such that we follow the [solidity style guide's order of functions](https://docs.soliditylang.org/en/latest/style-guide.html#order-of-functions)